README.md

```diff
@@ -0,0 +1,8 @@
+# Remote-Settings-Client
+Remote-Settings Client (read-only) built using Rust to fetch records, collections and comes with signature verification
+# Motivation
+We want to provide an easy way to access Remote-Settings data without having to reinvent the wheel and can provide more features saving both time and money
```

Comment:
I think this is superfluous since it's true for every library...

Motivations could be:
* Multiplatform 
* Robustness
* Genericity...
* Fun... :)